recipes/sbi/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "sbi" %}
+{% set version = "0.23.2" %}
+{% set python_min = "3.9" %}
```

Comment:
`python_min` is being added explicitly here to be ensure that it will match the lower bound of the upstream project if conda-forge's supported global pin range of Python 3.9 to Python 3.12 updates before the lower bound does.